scss/jquery/overrides/_tabs.scss

```diff
@@ -1,4 +1,4 @@
-/* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+/* stylelint-disable selector-max-compound-selectors, max-nesting-depth, selector-no-qualifying-type */
```

Comment:
Changed this a bit, applied it only inside `.crm-editable` block because
It is necessary here, because there are two buttons, and the only way to differentiate between them is by using attributes.